packages/astro/src/config/index.ts

```diff
@@ -35,7 +35,7 @@ export function getViteConfig(
 		]);
 		const logger = new Logger({
 			dest: nodeLogDestination,
-			level: 'info',
+			level: inlineAstroConfig.logLevel ?? 'info'
```

Comment:
Can we create the logger by importing `createNodeLogger` instead? That should be even accurate and we can create the logger like `const logger = createNodeLogger(inlineAstroConfig)`

https://github.com/withastro/astro/blob/c480b8f2c4a6282ccee706e9121ceacd91f5172b/packages/astro/src/core/config/logging.ts#L5-L12